src/preprocess.py

```diff
@@ -14,7 +14,7 @@
 opt = parser.parse_args()
 print(opt)
 
-assert opt == 2, 'Currently only two speakers are supported'
+assert opt != 2, 'Currently only two speakers are supported'
```

Comment:
The first one was right, we need the number of speakers to be 2. it should also be `opt.n_spkrs`

Reply:
Right! Super silly mistake from my end. Sorry about that, getting it fixed now